code/modules/clothing/suits/cardborg_suit.dm

```diff
@@ -0,0 +1,261 @@
+/*
+CONTENTS:
+1. Cardborg Helmets
+2. Cardborg Suits
+3. Disguise code
+*/
+
+/*
+ / Cardborg Helmets.
+*/
+/obj/item/clothing/head/cardborg
+	name = "cardborg helmet"
+	desc = "A helmet made out of a box."
+	// Extended description, currently shared with all the sub-types. Easy to add indvidual ones later if someone decides that.
+	var/extended_desc = "For reasons unknown to robotics experts across the galaxy, putting on a costume fashioned out of cardboard with some knobbly bits stuck on, \
+	some buttons drawn on in pen, and moving in a stereotypical 'robotic' fashion causes the wearer to be percieved as being an actual robot by other robots, \
+	cyborgs, and AI systems. The mechanism behind this is not understood, but they may use contextual information and other clues in order to see through the ruse. \
+	The wearer may also hallucinate themselves as being a robot as well."
+	icon_state = "cardborg_h"
+	item_state = "cardborg_h"
+	flags = BLOCKHAIR											// Robots don't have air (yet!).
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEEARS	// Robots don't wear masks or ear accessories (yet!).
+	flags_cover = HEADCOVERSEYES								// Robots don't wear glasses (yet!).
+	var/list/available_disguises = list("Standard") 			// All the sprites you can disguise as. A disguise will be randomly chosen from the list.
+	species_disguise = "High-tech robot"						// You appear to be this when examined instead of your mob's actual species.
+	dog_fashion = /datum/dog_fashion/head/cardborg 				// How this looks on Ian.
+	sprite_sheets = list(
+	"Grey" = 'icons/mob/clothing/species/grey/head.dmi',		// Greys have big heads and are smelly.
+	"Vox" = 'icons/mob/clothing/species/vox/head.dmi'			// Vox beaks stick out the normal helmets.
```

Comment:
This seems like a case of overcommenting to me, these variables should be explained on their setting on the parent type, not extra here